src/WebDriverBiDi/Session/CapabilitiesRequestInfo.cs

```diff
@@ -0,0 +1,70 @@
+// <copyright file="CapabilitiesRequest.cs" company="WebDriverBiDi.NET Committers">
+// Copyright (c) WebDriverBiDi.NET Committers. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// </copyright>
+
+namespace WebDriverBiDi.Session;
+
+using System.Text.Json.Serialization;
+
+/// <summary>
+/// Object containing capabilities requested when starting a new session.
+/// </summary>
+public class CapabilitiesRequestInfo
```

Comment:
To maintain compatibility with the spec, let's rename this class `CapabilityRequest`. Don't forget to match the name of the file in the copyright header.